src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -2228,7 +2260,32 @@ async def CommissionOnNetwork(self, nodeId: int, setupPinCode: int,
                     self.devCtrl, self.pairingDelegate, nodeId, setupPinCode, int(filterType), str(filter).encode("utf-8") if filter is not None else None, discoveryTimeoutMsec)
             )
 
-            return await asyncio.futures.wrap_future(ctx.future)
+            # If RCAC data is needed, await the result
```

Comment:
can this be pulled out into a separate function?

Reply:
I have moved setting the rcac_bytes variable to inside a new function get_rcac() to clean this up a little bit, it still gets returned in the OnNetworkCommission() here but a bit cleaner now.
Let me know if this will be acceptable.